lib/rules/lowercase_description.js

```diff
@@ -0,0 +1,94 @@
+const { isEmpty: _isEmpty, has: _has, keys: _keys } = require("lodash");
+const { format } = require("util");
+const Rule = require("./Rule");
+
+module.exports = class LowercaseDescription extends Rule {
+    static meta = {
+        ruleId: "lowercase_description",
+        message: "Expected lowercase description on keyword '%s'",
+        type: "off",
+        location: {},
+        hasFix: true,
+        // lintAfterFix: false,
+    };
+
+    static run(ast, config) {
+        if (_isEmpty(ast?.feature)) return [];
+
+        return new LowercaseDescription(ast, config).execute();
+    }
+
+    execute() {
+        this.checkDescription(this._ast);
+
+        return this.getProblems();
+    }
+
+    checkDescription(astObject) {
+        const keyword = _keys(astObject).shift();
+        if (["rule", "feature", "background", "scenario"].includes(keyword)) {
+            if (
+                typeof astObject[keyword].name === "string" &&
+                astObject[keyword].name.length !== 0
+            ) {
+                if (
+                    astObject[keyword].name[0] ===
+                    astObject[keyword].name[0].toUpperCase()
+                ) {
+                    const location = this.countKeywordDescriptionLine(
+                        this._ast,
+                        astObject[keyword].name
+                    );
```

Comment:
I think, the location info must be there already
```suggestion
                    const location = astObject[keyword].location
```

Reply:
astObject[keyword].location gives the location of keyword but we need the location of description of the keyword in order to correct it. 
e.g 
astObject[keyword].location gives { line: 11, column: 3 }
but the description is at location :  { line: 11, column: 13 }

Reply:
hmm, then I think this should be enough because `astObject[keyword].keyword` should contain `Scenario ` (with space at the end)
```js
const location = {...astObject[keyword].location}
location.column += astObject[keyword].keyword.length
```

Reply:
I was trying to be too accurate for column. There can be a space after scenario: like  `Scenario: Try` or there can't be space after scenario: like `Scenario:Try`. So to exactly locate the word "T", I used the above method.

Reply:
There could be another separate optional rule to require a single space after any `Feature:` `Scenario:` or `Scenario Outline:` - IMO most people will want a single space after the colon `:` character.

And a rule to require Feature "title" to exist - a feature file can;t just start with a line `Feature:` - there must be a "title" of the feature.

And a rule to require a description of every Scenario and Scenario Outline.

Maybe some of these rules already exist?

Reply:
No rules exists yet.
I just added two rules: (#17)
`require_title`
`space_after_keyword_colon`

Reply:
used the location as your suggestion @saw-jan dai